schedule_lessons/dashboard/static/dashboard/css/schedule_lesson.css

```diff
@@ -413,6 +413,65 @@ html body {
   margin: 10px 0;
 }
 
+.input-label-map {
+  display: block;
+  font-family: BrandonGrotesque-Regular;
```

Comment:
This entire CSS files is pretty messy. There are lots of styling additions, but I don't see why they are needed? There are no HTML elements that have these IDs or classes so I don't know why these stylings are here. Moreover, some stylings are duplicated like stylings for .slimField. Also the naming conventions for classes should be lowercase with a dash in between. One more thing is that the formatting for the stylings isn't great, there are randomly tabbed. But I think you can just remove all of these stylings because I don't see where they are being used.